core/orch_test.go

```diff
@@ -1320,8 +1352,36 @@ func TestPriceInfo_ReturnsBigRat(t *testing.T) {
 	expPricePerPixel = big.NewRat(20, 1)
 
 	priceInfo, err = orch.PriceInfo(ethcommon.Address{})
-	assert.Nil(t, err)
-	assert.Zero(t, expPricePerPixel.Cmp(big.NewRat(priceInfo.PricePerUnit, priceInfo.PixelsPerUnit)))
+	assert.Nil(err)
+	assert.Zero(expPricePerPixel.Cmp(big.NewRat(priceInfo.PricePerUnit, priceInfo.PixelsPerUnit)))
+	fixedPrice, err = common.PriceToFixed(expPricePerPixel)
+	expPrice = common.FixedToPrice(fixedPrice)
+	require.Nil(err)
```

Comment:
`require` should be before previous line.

Reply:
fixed in 1ff75f7